### PR TITLE
Tag ULID.jl v0.1.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [ararslan/ULID.jl](https://github.com/ararslan/ULID.jl)
Release: [v0.1.1](https://github.com/ararslan/ULID.jl/releases/tag/v0.1.1)
Travis: [Travis Build Status](https://travis-ci.org/ararslan/ULID.jl/builds/221802540)
Diff: [vs v0.1.0](https://github.com/ararslan/ULID.jl/compare/9d40eb0d7595db6f120a5e9c3d9a0dd4e2733ac4...8370bc19b45a981b08a77a204f1264fd85e58496)
`requires` vs v0.1.0: no changes
cc: @ararslan

### CODE REVIEW

File: ULID/versions/0.1.1/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.5-
```
Comment (tkelman) [inline ULID/versions/0.1.1/requires]: do you still need to support prereleases?

Comment (ararslan) [inline ULID/versions/0.1.1/requires]: Nope. This package is older than I remembered.

Comment (ararslan) [inline ULID/versions/0.1.1/requires]: Do I have to bump the minor version when dropping support for prereleases?

**Comment by ararslan** [comment]
Whoops, looks like attobot opened a new PR. I'll close this one.

File: ULID/versions/0.1.1/requires (hunk 1 above)
Comment (tkelman) [inline ULID/versions/0.1.1/requires]: nah

Comment (ararslan) [inline ULID/versions/0.1.1/requires]: Oh well, did it anyway. #yolo

**Pull request closed without merging**